.obsidian/plugins/fit/styles.css

```
@@ -0,0 +1,379 @@
+}
+
+.fit-avatar-container.error {
+    width: 16px;
+    height: 16px;
+    /* circle with cross */
+    content: url("data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIyNCIgaGVpZ2h0PSIyNCIgdmlld0JveD0iMCAwIDI0IDI0IiBmaWxsPSJub25lIiBzdHJva2U9ImN1cnJlbnRDb2xvciIgc3Ryb2tlLXdpZHRoPSIyIiBzdHJva2UtbGluZWNhcD0icm91bmQiIHN0cm9rZS1saW5lam9pbj0icm91bmQiIGNsYXNzPSJsdWNpZGUgbHVjaWRlLXgtY2lyY2xlIj48Y2lyY2xlIGN4PSIxMiIgY3k9IjEyIiByPSIxMCIvPjxwYXRoIGQ9Im0xNSA5LTYgNiIvPjxwYXRoIGQ9Im05IDkgNiA2Ii8+PC9zdmc+");
+    transform: scale(1.5);
+    aspect-ratio: 1;
+    padding: 1px;
+    margin-right: 4px;
+    margin: 4px;
```

Biome (2.1.2)

[error] Unexpected shorthand property margin after margin-right



(lint/suspicious/noShorthandPropertyOverrides)